js/login.js

```diff
@@ -0,0 +1,17 @@
+const loginBtn = document.getElementsByClassName('login_btn')[0];
+const idField = document.getElementsByClassName('id_field')[0];
+const pwField = document.getElementsByClassName('pw_field')[0];
+
+document.addEventListener('keyup', function() { 
+  //키 입력 후 발생되는 이벤트
+  if(idField.value.length > 0 && pwField.value.length > 0){
+      return loginBtn.style.backgroundColor = "#3897F0";
+  }
+  else {
+    return loginBtn.style.backgroundColor = "#C4E1FC";
+  }
```

Comment:
- 아래와 같이 작성하시면 중복되는 코드를 줄일 수 있습니다. 
- 단일 if else문일 때에는 '삼항연산자'를 활용해보세요!
```suggestion
  //키 입력 후 발생되는 이벤트
  const condition = idField.value.length > 0 && pwField.value.length > 0;
  loginBtn.style.backgroundColor = condition ? "#3897F0" : "#C4E1FC";
  }
```